<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateTab</class>
 <widget class="QWidget" name="TemplateTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Package Layout:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="packTree">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="file">
         <property name="text">
          <string>File Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="description"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addFile">
       <property name="text">
        <string>Add File</string>
       </property>
       <property name="icon">
        <iconset resource="../rc/rc.qrc">
         <normaloff>:/sourcefile/icon_set/icons/add.png</normaloff>:/sourcefile/icon_set/icons/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteSelection">
       <property name="text">
        <string>Delete Selection</string>
       </property>
       <property name="icon">
        <iconset resource="../rc/rc.qrc">
         <normaloff>:/sourcefile/icon_set/icons/delete.png</normaloff>:/sourcefile/icon_set/icons/delete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Test</string>
       </property>
       <property name="icon">
        <iconset resource="../rc/rc.qrc">
         <normaloff>:/icon_set/icons/eye.png</normaloff>:/icon_set/icons/eye.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rc/rc.qrc"/>
 </resources>
 <connections/>
</ui>
